### DISCUSSION TITLE
Nested objects for scope

### DISCUSSION THREAD

**Discussion by mackelito** [ngneat/transloco#717] [Q&A]
In feature libraries I load translations like this:
```
export const loginFeatureRoutes: Route[] = [
  {
    path: '',
    providers: [
      provideTranslocoScope({
        scope: 'login-feature',
        alias: 'login',
        loader,
      }),
    ],
  },
];
````

And ths strucure of my en.json is 
```
{
  "login-feature": {
    "header": "Hey!"
  },
}

```
it would be nice if I could pass like a nested scope 

```
{
  "login": {
    "feature" : {
	    "header": "Hey!"
	},
	"errors": {
		"not-allowed": "Nope, you should not be here"
	}
  },
}

// then use it like this in feature component:
    provideTranslocoScope({
      scope: 'login.feature', <---
      alias: 'login', <---
      loader,
    }),

// then use it like this in error component:
    provideTranslocoScope({
      scope: 'login.error', <---
      alias: 'error', <---
      loader,
    }),
```

**Comment by shaharkazaz** [comment]
You can still load the root scope and use the `prefix` property to get this behavior.
The scope's responsibility is for an entire translation file.

**Reply by swierzbicki** [reply]
> You can still load the root scope and use the prefix property to get this behavior.

It would be nice to give some example when you answer. Because i tried to find in the docs the `prefix` thing and i cannot find it or i did not understand what you wrote.
